requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
I've learned this from the hard way - using python 3.10 with pip-compile it removes necessary packages. So urge you to run this with python 3.9. Our envs are using 3.9.